koiic/vehicle-store-api
# #2 endpoint to create a new car

**What does this PR do?**

An endpoint to create a new car object

**Description of Task to be completed?**

As a user, I should be able to create a new car instance

**How should this be manually tested?**

- Fetch the branch with `git fetch origin ft-create-new-car-02 `
- Check out the branch with `git checkout ft-create-new-car-02 `
- Run the app: `docker-compose up` or `python manage.py runserver`

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

N/A

**Screenshots (if appropriate)**

N/A
